policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml

```
@@ -0,0 +1,19 @@
+description: MS provides a regular exemption of $6k for individuals, $12k for couples, $8k for heads of household.
```

Spell out states and avoid including values in descriptions.
```suggestion
description: Mississippi provides an exemption of this value depending on filing status.
```